modules/Microsoft.Compute/images/.test/common/deploy.test.bicep

```diff
@@ -0,0 +1,101 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for a testing purposes')
+@maxLength(90)
+param resourceGroupName string = 'ms.compute.images-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints')
+param serviceShort string = 'imgcom'
+
+// TODO: discuss the following challenge: roleassignment must be a globally unique identifier (GUID). The GUID is normally generated with role + scope + identity.
+// Identity in this case is the MSI deployed in the resourceGroupResources01 module. We cannot get that as output since the resource name requires a value that can be calculated at the start of the deployment.
+// Using the msi name as a workaround. Creating var in order not to duplicate its value (roleAssignment guid + input for resourceGroupResources01 module).
+// Same for destinationStorageAccountName. Creating other vars for consistency.
+
+// ========= //
+// Variables //
+// ========= //
+
+var managedIdentityName = 'dep-<<namePrefix>>-msi-${serviceShort}'
```

Comment:
Just as an open discussion: I'd personally prefer to have as few variables as possible (and hence specified the values in my PRs always directly on the module deployment whenever possible). Not saying it's wrong, but that I first started out with this too and eventually pivoted towards my current solution as deployments got bigger. Let's discuss this some time.

Reply:
I'd suggest to discuss that with the broader group. I'd agree if it wasn't the case, like for this module, that we sometimes need the same value in multiple places. If I had to choose between duplicating values and a bigger, yet more structured deployment I'd pick the latter.

Reply:
Should we still have the dedicated variables now that they are used only in a single place?

Reply:
Agree, will do. I'll also remove the `destinationStorageAccountName` parameter from dependencies' input, since it is not needed anymore.

Reply:
Done and retested